Docs/install tutorials (#232)
naturerobots/HSOS-SEP-PlantMap-2022: docs/getting-started/tutorials.md

## The main application :house_with_garden:

There are three main tabs: [`Dashboard`](#dashboard), [`Crops`](#crops) and [`3D`](#3d). Every tab consists of widgets.
Every widget can be toggled with the `widgets` button in the top right corner.
If you see an empty page, make sure some widgets are turned on.

### Changing current company and garden :arrows_clockwise:

In the application, first click on the profile picture in the top right corner, followed by `Administration`.
Select the company and garden you want and click on `save`.

### Dashboard :bar_chart:

<figure markdown>
  ![Dashboard](../imgs/dashboard.png){width=700px}
  <figcaption> Dashboard </figcaption>
</figure>

The dashboard contains a widget showing the current and forecasted weather and a map showing the current garden.
Additionally, you can see a list titled `Soil Parameter`, which displays soil data.
Hovering over a row highlights the location of the corresponding sensor on the map.

### Crops :seedling:

<figure markdown>
  ![Crops tab](../imgs/crops-tab.png){width=700px}
  <figcaption> Crops tab </figcaption>
</figure>

The crops tab consists of a table showing the beds of the current garden.

> Clicking on one of the beds shows the crops of the clicked bed in the table instead.

Reviewer: ```suggestion
Clicking on one of the beds shows it's crops.
```

To the right side of the table is a map of the current garden.
Green markers show the location of all the beds listed in the table and
hovering over one of them highlights the corresponding row.

> Clicking on one of the bed markers has the same effect as clicking on a bed in the table and
> shows the crops of the clicked bed.

Reviewer: ```suggestion
Clicking on one of the bed markers has the same effect as clicking on a bed in the table, the crops are shown.
```

When crops are shown in the table, their locations are marked on the map in the same way as the beds locations.
Like with the beds, hovering over them on the map highlights the associated row in the table.

For both beds and crops there is a button in the `3D` column to the right of the table.
For more information, see [3D](#3d).

### 3D

<figure markdown>
  ![3D tab](../imgs/3d-tab.png){width=700px}
  <figcaption> 3D tab </figcaption>
</figure>

The 3D tab has the same table and map as the [Crops](#crops) tab, the map is just larger.
Just like the Crops tab, beds and crops each have buttons in the `3D` column of the table.

Clicking the button brings you to a 3D model of the bed.
The model can be rotated with the left and moved with the right mouse button.
The scroll wheel can be used to zoom.

> If you clicked on the `3D` button for a crop, an overlay will be shown,
> framing the crop and providing additional information about it.

Reviewer: ```suggestion
If you clicked on the `3D` button for a crop, an overlay with information will be shown. 
```
